connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginScannerTest.java

```diff
@@ -145,10 +148,28 @@ public void testScanningMixOfValidAndInvalidPlugins() throws Exception {
         assertEquals(expectedClasses, classes);
     }
 
+    @Test
+    public void testScannedPluingsForUndefinedVersion() {
+        PluginScanResult unversionedPluginsResult = scan(filterPluginsResourceDir(TestPlugins.pluginPath(), TestPlugins.TestPlugin.SAMPLING_HEADER_CONVERTER.resourceDir()));
+        assertFalse(unversionedPluginsResult.isEmpty());
+        unversionedPluginsResult.forEach(pluginDesc -> assertEquals(PluginDesc.UNDEFINED_VERSION, pluginDesc.version()));
+    }
+
+    @Test
+    public void testScannedPluingsForVersion() {
+        PluginScanResult versionedPluginResult = scan(filterPluginsResourceDir(TestPlugins.pluginPath(), TestPlugins.TestPlugin.READ_VERSION_FROM_RESOURCE_V1.resourceDir()));
```

Comment:
```suggestion
        PluginScanResult versionedPluginResult = scan(TestPlugins.pluginPath(TestPlugins.TestPlugin.READ_VERSION_FROM_RESOURCE_V1));
```

Reply:
Thanks Manually committed to add TestPlugins dependency workaround